docs/en/connector-v2/source/GraphQL.md

```diff
@@ -0,0 +1,188 @@
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [x] [stream](../../concept/connector-v2-features.md)
+- [ ] [parallelism](../../concept/connector-v2-features.md)
+
+## Options
+
+| name                        | type    | required | default value           |
+| --------------------------- | ------- | -------- | ----------------------- |
+| url                         | String  | Yes      | -                       |
+| query                       | String  | Yes      | -                       |
+| variables                   | Config  | No       | -                       |
+| enable_subscription         | boolean | No       | false                   |
+| timeout                     | Long    | No       | -                       |
+| content_field               | String  | Yes      | $.data.{query_object}.* |
+| schema.fields               | Config  | Yes      | -                       |
+| format                      | String  | No       | json                    |
+| params                      | Map     | Yes      | -                       |
+| poll_interval_millis        | int     | No       | -                       |
+| retry                       | int     | No       | -                       |
+| retry_backoff_multiplier_ms | int     | No       | 100                     |
+| retry_backoff_max_ms        | int     | No       | 10000                   |
+| enable_multi_lines          | boolean | No       | false                   |
+| common-options              | config  | No       | -                       |
+
+### url [String]
+
+http request url
+
+### query [String]
+
+GraphQL expression query string
+
+### variables [String]
+
+GraphQL Variables
+
+for example 
+
+```
+variables = {
+   limit = 2
+}
+```
+
+### enable_subscription [boolean]
+
+1. true :  Build a socket reader to subscribe to the GraphQL service
+2. false :  Build an http reader subscription to the GraphQL service
+
+### timeout [Long]
+
+Time-out Period
+
+### content_field [String]
+
+JSONPath wildcard
+
+### params [Map]
+
+http request params
+
+### poll_interval_millis [int]
+
+request http api interval(millis) in stream mode
+
+### retry [int]
+
+The max retry times if request http return to `IOException`
+
+### retry_backoff_multiplier_ms [int]
+
+The retry-backoff times(millis) multiplier if request http failed
+
+### retry_backoff_max_ms [int]
+
+The maximum retry-backoff times(millis) if request http failed
+
+### format [String]
+
+the format of upstream data, default `json`.
+
+### schema [Config]
+
+Fill in a fixed value
+
+```hocon
+    schema = {
+        fields {
+            metric = "map<string, string>"
+            value = double
+            time = long
+            }
+        }
+
+```
+
+#### fields [Config]
+
+the schema fields of upstream data
+
+### common options
+
+Source plugin common parameters, please refer to [Source Common Options](../source-common-options.md) for details
+
+## Example
+
+### Query:
```

Comment:
Hi @eyys . After #9159 , we can't add `:` in header. Could you raise a PR to remove it to fix ci?